e2e/pages/uploadDocuments/uploadDocuments.js

```diff
@@ -0,0 +1,31 @@
+const I = actor();
+
+module.exports = {
+
+	documents: {
+		socialWorkChronology: '#documents_socialWorkChronology_document_uploadDocuments_typeOfDocument',
+		socialWorkStatementAndGenogram: '#documents_socialWorkStatement_document_uploadDocuments_typeOfDocument',
+		socialWorkAssessment: '#documents_socialWorkAssessement_document_uploadDocuments_typeOfDocument',
+		carePlan: '#documents_socialWorkCarePlan_document_uploadDocuments_typeOfDocument',
+	},
+
+	fields: {
+		socialWorkChronologyStatus: '#documents_socialWorkChronology_document_uploadDocuments_documentStatus',
+		socialWorkChronologyReason: '#documents_socialWorkChronology_document_uploadDocuments_statusReason',
+		socialWorkStatementAndGenogramStatus: '#documents_socialWorkStatement_document_uploadDocuments_documentStatus',
+		socialWorkAssessmentStatus: '#documents_socialWorkAssessement_document_uploadDocuments_documentStatus',
+		carePlanStatus: '#documents_socialWorkCarePlan_document_uploadDocuments_documentStatus',
+
+	},
+
+	uploadDocuments(file) {
```

Comment:
The way I see page objects is that they are an interface for the page. Since that for me it makes sense to split API into methods that do perform granular actions available on the page.

I would personally have the following methods:

```
- uploadSocialWorkChronology
- uploadSocialWorkStatementAndGenogram
- uploadSocialWorkAssessment
- uploadCarePlan
```

This way it is more descriptive in terms of what test is doing and more reusable.

Something to consider in the future.

Reply:
In fact we could have:

```
- uploadSocialWorkChronology
- uploadSocialWorkStatement
- uploadSocialWorkAssessment
- uploadSocialWorkCarePlan
```

to match names in domain model.